### DISCUSSION TITLE
Enquiry about FragPipe Analyst for Phosphoproteome Analysis

### DISCUSSION THREAD

**Discussion by TeckYewLow** [Nesvilab/FragPipe-Analyst-doc#40] [Ideas]
Hi,

(1) I have a phosphoproteomics dataset generated from tims-TOF in DA mode, and I have analyzed the raw data with MS Fragger. I am planning to use FragPipe for LFQ (peptide) analysis. In Fragpiple Analyst, I chose LFQ (peptide) for the Data Type and MaxLFQ for the Intensity Type.

When I chose "Max LFQ" I got the following error message: u"The labels/'run names' in the experimental design DID NOT match with lfq column names in maxquants proteinGroups file Run LFQ-Analyst with correct labels in the experimental design". However, when I switched to using "Intensity" for Intensity Type, then it ran fine. I am uploading my combined_phosphopeptide. tsv and the experimental annotation file for your perusal.
[combined_phosphopeptide.txt](https://github.com/user-attachments/files/16154203/combined_phosphopeptide.txt)
[experiment_annotation.txt](https://github.com/user-attachments/files/16154250/experiment_annotation.txt)

(2) A second problem I encountered is the phosphopeptide annotation. For example, my MSFragger results shows phosphopeptides from MSH protein as:

Modified Sequence Prev AA Next AA Start End Peptide Length Charges Assigned Modifications Protein
AAAAPGAS[79.9663]PSPGGDAAWSEAGPGPRPLAR R S 34 62 29 3 8S(79.9663) sp|P52701|MSH6_HUMAN
AAAAPGASPS[79.9663]PGGDAAWSEAGPGPRPLAR R S 34 62 29 3 10S(79.9663) sp|P52701|MSH6_HUMAN
AAAAPGASPSPGGDAAWS[79.9663]EAGPGPRPLAR R S 34 62 29 3 18S(79.9663) sp|P52701|MSH6_HUMAN

However, after Fragpipe Analyst analysis:

1 | P52701_AAAAPGASPSPGGDAAWSEAGPGPRPLAR.1 | P52701 | MSH6 |
2 | P52701_AAAAPGASPSPGGDAAWSEAGPGPRPLAR.2 | P52701 | MSH6 |

The phosphopeptides were appended with xxx.1 or xxx.2. How could we know the phosphorylated isoforms for each peptide?

Thank you so much!

Teck

**Comment by hsiaoyi0504** [comment]
@TeckYewLow We don't officially support PTM peptides yet, but thanks for reporting your observation and we will consider it when implementing related features.
